opendbc/safety/modes/nissan.h

```diff
@@ -67,6 +67,7 @@ static bool nissan_tx_hook(const CANPacket_t *to_send) {
       {0., 5., 15.},
       {5., 3.5, .4}
     },
+    .frequency = 100U,  // 100 Hz
```

Comment:
don't duplicate the value in the comment. and ideally put the unit in the variable name if it's unclear, but in this case, i think it's clear enough